<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmRepairModbusConnection</class>
 <widget class="QMainWindow" name="FrmRepairModbusConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Thông tin</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Thời gian quét (ms)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Số thanh ghi</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Địa chỉ bắt đầu</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_StartRegister">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>40001</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_TimerScan">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_NumberRegister">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_typeRegister">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Holding Registers</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Input Registers</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Loại thanh ghi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Start">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>32</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Bắt đầu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Thời gian lỗi (ms)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_TimerOut">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Hex">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>40</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Hex</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Dec">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>40</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Dec</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>981</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>981</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Đánh giá</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Đếm:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Count">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lỗi quét:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ErrorScan">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="spinBox_displayTo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>145</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>148</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Đến:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_displayFrom">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>145</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>148</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hiển thị thanh ghi từ:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_typeRegister</tabstop>
  <tabstop>spinBox_StartRegister</tabstop>
  <tabstop>spinBox_NumberRegister</tabstop>
  <tabstop>spinBox_TimerOut</tabstop>
  <tabstop>spinBox_TimerScan</tabstop>
  <tabstop>radioButton_Hex</tabstop>
  <tabstop>radioButton_Dec</tabstop>
  <tabstop>pushButton_Start</tabstop>
  <tabstop>plainTextEdit_Result</tabstop>
  <tabstop>spinBox_displayFrom</tabstop>
  <tabstop>spinBox_displayTo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
